<def f='include/c++/9/future' l='99' ll='102' type='void std::future_error::future_error(std::future_errc __errc)'/>
<use f='include/c++/9/future' l='438' u='c' c='_ZNSt13__future_base13_State_baseV216_M_break_promiseESt10unique_ptrINS_12_Result_baseENS2_8_DeleterEE'/>
